internal/builders/golang/build.go

```diff
@@ -119,6 +119,13 @@ func (*Builder) Build(ctx *context.Context, build config.Build, options api.Opti
 	if err := run(ctx, cmd, env, build.Dir); err != nil {
 		return errors.Wrapf(err, "failed to build for %s", options.Target)
 	}
+	if ctx.Config.Deterministic {
+		// could do this always, since the Date is Now or CommitDate upstream
```

Comment:
I would vote for doing this always... maybe add a config like `builds.time` set by default to `{{ .Timestamp }}`, then the user can change to `{{ .CommitDate }}`? wdyt?

Reply:
Done, called it `ModTimestamp`, as its the modified time for the file, and its in the unix timestamp format.